src/plugins/Wheel.ts

```diff
@@ -110,6 +110,7 @@ export class Wheel extends Plugin
     {
         super(parent);
         this.options = Object.assign({}, DEFAULT_WHEEL_OPTIONS, options);
+        this.parent._wheelAxis = this.options.axis;
```

Comment:
This does not have to be in the wheel at all as it's not a wheel-specific concept.